src/hooks/usePaginate.ts

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react"
+
+export const usePaginate = (
+  initialPage = 0
+): [number, (page: number) => void] => {
+  const [curPage, setCurPage] = useState(initialPage)
+
+  return [
+    curPage + 1,
```

Comment:
Hmm is there a reason for this offset? 
worried that it might be confusing to expect a currPage to be = 1 after the caller calls setCurrPage(2)

Reply:
`Pagination` component is offset by 1.